lib/offline/storage.js

```diff
@@ -400,14 +401,14 @@ shaka.offline.Storage = class {
       const clearKeyDataLicenseServerUri = manifest.variants.some((v) => {
         if (v.audio) {
           for (const drmInfo of v.audio.drmInfos) {
-            if (!drmInfo.licenseServerUri.startsWith('data:')) {
+            if (drmInfo.licenseServerUri.startsWith('data:')) {
```

Comment:
So the logic of this `some` call is being inverted. Was it in error before, or is this a change in logic? Returning true if there is a "data" license server URI does seem to fit the variable name `clearKeyDataLicenseServerUri` better, I guess...

Reply:
This logic and the variable it computes were just added Dec 31.  This hasn't been in a release yet.

Reply:
Yes :)